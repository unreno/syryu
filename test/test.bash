#!/usr/bin/env bash


failures=0

echo "Testing amino_acid_counter.bash"
../scripts/amino_acid_counter.bash test.fasta

echo "Diffing output. They should be identical."
diff test.amino_acid_counts.tsv test.amino_acid_counts.expected.tsv

failures=`expr $failures + $?`

\rm test.amino_acid_counts.tsv

echo
echo "Failures: $failures"
