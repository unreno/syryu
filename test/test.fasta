>1433B_HUMAN P31946 14-3-3 protein beta/alpha
MTMDKSELVQKAKLAEQAERYDDMAAAMKAVTEQGHELSNEERNLLSVAYKNVVGARRSS
WRVISSIEQKTERNEKKQQMGKEYREKIEAELQDICNDVLELLDKYLIPNATQPESKVFY
LKMKGDYFRYLSEVASGDNKQTTVSNSQQAYQEAFEISKKEMQPTHPIRLGLALNFSVFY
YEILNSPEKACSLAKTAFDEAIAELDTLNEESYKDSTLIMQLLRDNLTLWTSENQGDEGD
AGEGEN
>1433E_HUMAN P62258 14-3-3 protein epsilon
MDDREDLVYQAKLAEQAERYDEMVESMKKVAGMDVELTVEERNLLSVAYKNVIGARRASW
RIISSIEQKEENKGGEDKLKMIREYRQMVETELKLICCDILDVLDKHLIPAANTGESKVF
YYKMKGDYHRYLAEFATGNDRKEAAENSLVAYKAASDIAMTELPPTHPIRLGLALNFSVF
YYEILNSPDRACRLAKAAFDDAIAELDTLSEESYKDSTLIMQLLRDNLTLWTSDMQGDGE
