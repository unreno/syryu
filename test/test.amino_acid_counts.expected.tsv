ProteinName	ACount	BCount	CCount	DCount	ECount	FCount	GCount	HCount	ICount	JCount	KCount	LCount	MCount	NCount	OCount	PCount	QCount	RCount	SCount	TCount	UCount	VCount	WCount	XCount	YCount	ZCount	Length
"1433B_HUMAN P31946 14-3-3 protein beta/alpha"	21	0	2	14	30	6	10	2	10	0	20	24	8	14	0	5	15	10	19	12	0	11	2	0	11	0	246
"1433E_HUMAN P62258 14-3-3 protein epsilon"	24	0	3	20	25	5	10	3	13	0	17	26	10	9	0	5	6	14	14	10	0	12	2	0	12	0	240
